src/Symfony/Component/Security/Core/Authorization/Voter/ExpressionVoter.php

```diff
@@ -85,7 +85,7 @@ private function getVariables(TokenInterface $token, $object)
         $variables = array(
             'token' => $token,
             'user' => $token->getUser(),
-            'object' => $object,
+            'object' => $subject,
```

Comment:
The `object` variable should probably be renamed to `subject` too. But that should be done in `2.8` and we should throw a deprecation when accessing `object`. Would that actually be possible?